include.xml

```diff
@@ -2,11 +2,12 @@
 <project>
 	<icon path="assets/images/logo/HaxeFlixel.svg" />
 	
-	<assets path="assets/sounds" include="*.ogg" if="html5" unless="unit-test" />
+	<assets path="assets/sounds" include="*.m4a" if="html5" unless="unit-test" />
 	<assets path="assets/sounds" include="*.mp3" if="flash" embed="true" />
-	<assets path="assets/sounds" include="*.ogg" unless="flash || html5" embed="true" />
+	<assets path="assets/sounds" include="*.ogg" if="native" embed="true" />
+	
 	<assets path="assets/fonts/nokiafc22.ttf" embed="true" />
-	<assets path="assets/fonts/arial.ttf" embed="true" unless="FLX_NO_DEBUG" />
+	<assets path="assets/fonts/montserrat-Regular.ttf" embed="true" if="debug" />
```

Comment:
Looks correct to me (unless you meant it as a joke):
http://www.fontsquirrel.com/fonts/montserrat

I think it's meant to be "Mont Serrat" as opposed to "Monster Rat"